README.md

```diff
@@ -43,12 +45,14 @@ which are needed for building the package.
 Checks if there is a supported issue tracker reference in the changelog when
 version in the .spec file has been changed.
 
-### check:commited
-Checks if all changes to the local git repository are commited.
+### check:committed
+Checks if all changes to the local git repository are committed.
 It doesn't check if changes
 are sent to a remote git repository. Its main intention is to ensure that all
 changes are tracked before making a package.
 
+Note: The check is skipped when environment variable `COMMIT_CHECK` is set to `0`.
```

Comment:
What about naming the option after the task, `CHECK_COMMITTED` ?

Reply:
It turned out that we need to keep the check for *new* files because `rake package` archives only the files known to Git, so the package build would miss them.

So we can only skip the check for modified files. But that would be still sufficient in many situations.

Reply:
Let's use `CHECK_MODIFIED` instead...